pom.xml

```diff
@@ -65,9 +65,9 @@ under the License.
   <properties>
     <!-- Because Checkstyle 7+ requires Java 8 -->
     <javaVersion>8</javaVersion>
-    <mavenVersion>3.0</mavenVersion>
+    <mavenVersion>3.6.3</mavenVersion>
```

Comment:
please do not update the minimum required Maven version to the latest available maven. Currently, our minimum should be 3.1.1. Please also do not update everything in one ticket/PR.

Reply:
Done, I had to add some dependencies (sisu) to make it work.